pkg/oktaapi/client_wrapper.go

```diff
@@ -110,7 +110,11 @@ func (cw *ClientWrapper) FetchUserInfos(ctx context.Context, usernames []string)
 	searchedUsers, _, err := cw.oktaClient.User.ListUsers(ctx, search)
 
 	// Log response time information
-	logger.Info("FetchUserInfos okta call", zap.Int64("response-time-ms", time.Since(start).Milliseconds()))
+	logger.Info("FetchUserInfos okta call",
+		zap.Int64("response-time-ms", time.Since(start).Milliseconds()),
+		zap.Int("username-count", len(usernames)),
+		zap.Int("user-count", len(searchedUsers)),
```

Comment:
Just wanted to add these two fields so we can see if there's a correlation between the number of users searched, returned, and the response time itself